srcpkgs/azote/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'azote'
 pkgname=azote
-version=1.7.10
-revision=2
+version=1.8.1
+revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools"
 depends="python3>=3.4 python3-setuptools python3-gobject python3-Pillow gtk+3 feh python3-send2trash xrandr grim slurp ImageMagick python3-yaml"
```

Comment:
Please drop the following packages from the dependencies:
- [ ] xrandr
- [ ] feh
- [ ] python3-send2trash
- [ ] grim
- [ ] slurp
- [ ] ImageMagick
- [ ] python3-yaml
- [ ] python3-setuptools

All but the last enable optional functionality, and the program itself provides a warning about missing packages and disabled functionality when users run without these packages installed. There is no need to pull in everything by force, especially the `xrandr` dependency for Wayland users, or the `grim` and `slurp` dependencies for Xorg users.

The last dependency is strictly wrong, as `setuptools` is not needed anywhere for runtime operation of `azote`.

Reply:
Updated, thanks!

Reply:
Well, azote uses `pkg_resources` module from setuptools.

Reply:
My mistake, `python3-setuptools` definitely belongs in `depends` because of `pkg_resources`. Thanks @Chocimier. 

This means `python3` can just be dropped, because `python3-setuptools` already pulls that in.

Reply:
Updated, thanks @Chocimier and @ahesford!